client/browser-app/package.json

```diff
@@ -3,33 +3,41 @@
   "name": "browser-app",
   "version": "0.2.0",
   "dependencies": {
+    "@eclipse-emfcloud/ecore-glsp-common": "0.2.0",
     "@eclipse-emfcloud/ecore-property-view": "0.2.0",
     "@eclipse-emfcloud/ecore-tree-editor": "0.2.0",
     "@eclipse-emfcloud/theia-ecore": "0.2.0",
-    "@theia/core": "^1.12.1",
-    "@theia/editor": "^1.12.1",
-    "@theia/filesystem": "^1.12.1",
-    "@theia/markers": "^1.12.1",
-    "@theia/messages": "^1.12.1",
-    "@theia/monaco": "^1.12.1",
-    "@theia/navigator": "^1.12.1",
-    "@theia/plugin-ext-vscode": "^1.12.1",
-    "@theia/preferences": "^1.12.1",
-    "@theia/process": "^1.12.1",
-    "@theia/property-view": "^1.12.1",
-    "@theia/terminal": "^1.12.1",
-    "@theia/workspace": "^1.12.1"
+    "@theia/core": "^1.0.0",
```

Comment:
I don't know about the ModelServer and TreeEditor, but I'm pretty sure GLSP doesn't support any older version of Theia. Is there a specific reason to relax the Theia dependency versions? (It doesn't really matter, since GLSP will add more constraints anyway; but I'm curious about the change)

Reply:
I aligned it with both `eclipse-glsp` and `eclipse-emfcloud` projects where we use the relaxed version of Theia instead of a locked one (e.g. https://github.com/eclipse-glsp/glsp-theia-integration/blob/master/examples/browser-app/package.json).